### PR TITLE
add Dockerfile for container providing easy access to EESSI pilot repository

### PR DESCRIPTION
**Pull request by boegel**
Doesn't include `fuse-overlayfs`, since pure clients don't really need that.

### CODE REVIEW

**Comment by boegel** [comment]
@bedroge Let's merge this, container for clients that want to play with the pilot repo has proved to be working well?

File: containers/README [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Docker container to provide easy access to EESSI pilot repository,
+see https://hub.docker.com/repository/docker/eessi/client-pilot and https://eessi.github.io/docs/pilot .
```
Comment (bedroge) [inline containers/README:2]: You will really have to make my day first ;-)

```suggestion
see https://hub.docker.com/repository/docker/eessi/client-pilot and https://eessi.github.io/docs/pilot.
```

**Review by bedroge — CHANGES_REQUESTED** [review]
Looks good to me, except that I have another PR (#36) which also adds files to this containers dir. That PR uses a README.md, though. Can we try making them compatible, by for instance renaming your README to README.md and describing this container in a specific section?

**Comment by boegel** [comment]
@bedroge I think this is good to go now...

File: containers/Dockerfile.EESSI-client-pilot-centos7 [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM docker.io/library/centos:7.8.2003
+
+RUN yum install -y http://cvmrepo.web.cern.ch/cvmrepo/yum/cvmfs-release-latest.noarch.rpm \
+ && yum install -y cvmfs cvmfs-config-default cvmfs-fuse3 sudo vim openssh-clients \
+ && curl -OL https://github.com/EESSI/filesystem-layer/releases/download/v0.2.1/cvmfs-config-eessi-0.2.1-1.noarch.rpm \
+ && yum install -y cvmfs-config-eessi-0.2.1-1.noarch.rpm
```
Comment (bedroge) [inline containers/Dockerfile.EESSI-client-pilot-centos7:6]: I think we don't need to download this rpm manually (also prevents it from being stored in the root, which is annoying for autocompletion ;-) )?
```suggestion
 && yum install -y https://github.com/EESSI/filesystem-layer/releases/download/v0.2.1/cvmfs-config-eessi-0.2.1-1.noarch.rpm 
```

File: containers/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Containers for EESSI
+
+This directory contains recipes for containers that are useful in the scope of the EESSI project.
+
+## Client container
+
+Container to provide easy access to EESSI pilot repository,
+see https://hub.docker.com/repository/docker/eessi/client-pilot and https://eessi.github.io/docs/pilot .
```
Comment (bedroge) [inline containers/README.md:8]:
```suggestion
see https://hub.docker.com/repository/docker/eessi/client-pilot and https://eessi.github.io/docs/pilot.
```
I don't like spaces, make my day again! :-D

**Review by bedroge — CHANGES_REQUESTED** [review]

**Review by bedroge — APPROVED** [review]
Haven't been able to test the Arm image on my RPi, but it looks good to me.

**Pull request merged**